src/com/itacademy/eshop/Main.java

```diff
@@ -1,9 +1,10 @@
 package com.itacademy.eshop;
 
+import com.itacademy.eshop.exceptions.DuplicateProductException;
 import com.itacademy.eshop.exceptions.ProductNotFoundException;
 
 public class Main {
-    public static void main(String[] args) throws ProductNotFoundException {
+    public static void main(String[] args) throws ProductNotFoundException, DuplicateProductException {
```

Comment:
main should not throw exceptions, preferably all exceptions should be caught before